### PR TITLE
better clarify namespace precendence on resource manipulation; clarif…

### PR DESCRIPTION
**Pull request by gabemontero**
…y role of two repos

@wewang58 and @bparees - please review doc updates based on recent discussion we have had.

@dobbymoodge - I'll use this PR to test the changes we did recently on ci.openshift

### CODE REVIEW

**Comment by gabemontero** [comment]
@dobbymoodge - the ci.openshift change looked to me to have worked.  Everything I was expecting seems to have occurred, but take a peek from your end and see if you agree.

File: README.md [hunk 1]
```diff
@@ -47,11 +50,11 @@ A series of Jenkins "build step" implementations are provided, which you can sel
 
 8. "Create OpenShift Resource(s)":  performs the equivalent of an `oc create` command invocation; this build step takes in the provided JSON or YAML text, and if it conforms to OpenShift schema, creates whichever OpenShift resources are specified.
 
-	- NOTE: if a namespace is specified for any of the resources, the plugin will attempt to create the resource in that namespace instead of the namespace specified in the "The name of the project to create the resources in" field.  However, the authorization token provided must have edit permission to that project/namespace.
+	- NOTE: if a namespace is specified in the provided json/yaml for any of the resources, that namespace will take precedence over the namespace specified in the "The name of the project to create the resources in" field.  However, the authorization token provided in the field "The authorization token for interacting with OpenShift" must have edit permission to any project/namespace reference in the json/yaml.  Or if no token is specified, the assumption is that Jenkins in running in OpenShift, and the default service account associated with the project/namespace Jenkins is running in must have edit permission.
```
Comment (bparees) [inline README.md]: "permission to any project/namespace reference in the json/yaml."

referenced.

File: README.md [hunk 2]
```diff
@@ -17,8 +17,11 @@ is already provided and can be invoked from your Jenkins jobs, or if you are not
 
 NOTE:  This plugin requires JDK 1.8, based on its maven dependency openshift-restclient-java.
 
-This document represents the most technical, detailed description of this plugin.  For reference, the associated wiki page for this plugin on the Jenkins site is [here](https://wiki.jenkins-ci.org/display/JENKINS/OpenShift+Pipeline+Plugin).
-The most important piece of information on that page will be the indication of the latest officially released version of this plugin.
+The documentation and code at [https://github.com/openshift/jenkins-plugin](https://github.com/openshift/jenkins-plugin) always hosts the very latest version, including possibly pre-released versions that are still under test.
+The associated repository under the JenkinsCI project, [https://github.com/jenkinsci/openshift-pipeline-plugin](https://github.com/jenkinsci/openshift-pipeline-plugin), only is updated as part of cutting
```
Comment (bparees) [inline README.md]: s/only is updated/is only updated/

File: README.md (hunk 1 above)
Comment (bparees) [inline README.md]: s/Or if no token/If no token/

File: README.md [hunk 3]
```diff
@@ -47,11 +50,11 @@ A series of Jenkins "build step" implementations are provided, which you can sel
...
 
-	- NOTE: if a namespace is specified for any of the resources when deleting via JSON/YAML, the plugin will attempt to delete the resource in that namespace instead of the namespace specified in the "The name of the project to delete the resources in" field.  However, the authorization token provided must have edit permission to that project/namespace.
+	- NOTE: if a namespace is specified in the provided json/yaml for any of the resources, that namespace will take precedence over the namespace specified in the "The name of the project to create the resources in" field.  However, the authorization token provided in the field "The authorization token for interacting with OpenShift" must have edit permission to any project/namespace reference in the json/yaml.  Or if no token is specified, the assumption is that Jenkins in running in OpenShift, and the default service account associated with the project/namespace Jenkins is running in must have edit permission.
```
Comment (bparees) [inline README.md]: same nits here

**Comment by bparees** [comment]
couple nits, otherwise lgtm

File: README.md (hunk 1 above)
Comment (gabemontero) [inline README.md]: part of next push

File: README.md (hunk 2 above)
Comment (gabemontero) [inline README.md]: part of next push

File: README.md (hunk 3 above)
Comment (gabemontero) [inline README.md]: yep

**Comment by gabemontero** [comment]
comments from @bparees incorporated - thanks

@wewang58 - any chance you'll be able to confirm if these updates make the behavior clearer to you?
I'll wait for a bit, see if you have a chance, before merging this.

Also, we'll keep the motivating issue open to sort through your attempts create resources in varying namespaces.

**Comment by bparees** [comment]
lgtm

**Comment by gabemontero** [comment]
thx @bparees

**Pull request merged**